app/views/passengers/index.haml

```diff
@@ -24,7 +24,7 @@
           %td.inactive Inactive
           %td.no_note No Note
 - if @passengers.present?
-  %table#passengers{class: passengers_table_class}
+  %table.passengers
```

Comment:
Any particular reason for the change from id to class?

Reply:
In particular, there's still CSS that targets `table#passengers {`

Reply:
I guess since there was more than just one now that I've added (in another PR but still) the archived passenger page that uses the same kind of table, I felt weird about having it be an ID. BUT, that rule may in fact only apply to divs on the same page as each other. So I'll change it back (and not changing the css was a massive oversight).